### PR TITLE
functions added.

### PR DESCRIPTION
**Pull request by emrullahardc**

### CODE REVIEW

File: src/Helpers/AliasStatus.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Visiosoft\SiteModule\Helpers;
+
+use Visiosoft\SiteModule\SiteModuleInterface;
+
+abstract class AliasStatus
+{
+    const WAITING = 0;
```
Comment (fatihalp) [inline src/Helpers/AliasStatus.php:9]: good abstract class @emrullahardc 

@spektra2147

File: src/Helpers/AliasStatus.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const CREATED = 1;
+    const CREATE_FAIL = 2;
+    const SSL_FAIL = 3;
```
Comment (fatihalp) [inline src/Helpers/AliasStatus.php:12]: I'm not sure Site module is responsible to SSL

File: src/Jobs/NewSiteSSH.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $serverPassword = $this->server->getPassword();
+
+        $ssh = new SSH2($this->server->getIp(), 22);
+        $ssh->login('pure', $serverPassword);
```
Comment (fatihalp) [inline src/Jobs/NewSiteSSH.php:41]: there are 6 duplicate of logging server. Can you create loginAndRunCommand function inside server module

Comment (fatihalp) [reply src/Jobs/NewSiteSSH.php:41]: to centralize

File: src/Helpers/AliasStatus.php (hunk 1 above)
Comment (emrullahardc) [reply src/Helpers/AliasStatus.php:9]: thank you.

File: src/Helpers/AliasStatus.php (hunk 2 above)
Comment (emrullahardc) [reply src/Helpers/AliasStatus.php:12]: You are right. I will remove

File: src/Jobs/NewSiteSSH.php (hunk 3 above)
Comment (emrullahardc) [reply src/Jobs/NewSiteSSH.php:41]: Sure

**Pull request merged**